packages/runtime-core/src/h.ts

```diff
@@ -192,7 +194,7 @@ export function h(type: any, propsOrChildren?: any, children?: any): VNode {
   } else {
     if (l > 3) {
       children = Array.prototype.slice.call(arguments, 2)
-    } else if (l === 3 && isVNode(children)) {
+    } else if (l === 3 && (isVNode(children) || isRef(children))) {
```

Comment:
It seems this is a breaking change

Reply:
This conditional statement just turns `children` into an array